src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/DeprecationTest.php

```diff
@@ -49,9 +50,110 @@ public function testItRulesOutFilesOutsideVendorsAsIndirect()
         $this->assertFalse($deprecation->isIndirect());
     }
 
+    public function providerIsSelf(): array
+    {
+        return [
+            [true, '', 'MyClass1', ''],
```

Comment:
Please add a key with a description for each case, it will make things easier to understand when tests fail: `"some description"` =>              [true, '', 'MyClass1', '']

Reply:
Done, thank you :)